modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -5,7 +5,7 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="create-a-kubeletconfig-crd-to-edit-kubelet-parameters_{context}"]
```

Comment:
It might be out of your purview, but it looks like `cnf-configuring-kubelet-config-nro.adoc` also needs its title to be revised in a similar way.